docs/source/release/v6.0.0/diffraction.rst

```diff
@@ -5,115 +5,124 @@ Diffraction Changes
 .. contents:: Table of Contents
    :local:
 
-.. warning:: **Developers:** Sort changes under appropriate heading
-    putting new features at the top of the section, followed by
-    improvements, followed by bug fixes.
-
-New features
-------------
-
 Powder Diffraction
 ------------------
-New features
+
+New Algorithms
+##############
+
+- :ref:`PaalmanPingsAbsorptionCorrection <algm-PaalmanPingsAbsorptionCorrection>` uses a numerical integration method to calculate attenuation factors for all Paalman Pings terms
+- :ref:`LeadPressureCalc <algm-LeadPressureCalc>` to calculate the pressure of a sample given the d-Spacing of the Lead (111) peak and the sample temperature
+- :ref:`CorelliPowderCalibrationDatabase <algm-CorelliPowderCalibrationDatabase>` to save CORELLI geometry calibration to database
+- :ref:`CorelliPowderCalibrationApply <algm-CorelliPowderCalibrationApply>` to apply a CORELLI calibration table to CORELLI EventWorkspace
+- :ref:`CorelliPowderCalibrationCreate <algm-CorelliPowderCalibrationCreate>` adjusts the position and orientation of CORELLI banks in order to optimize the comparison of observed peaks to reference data
+- :ref:`ClipPeaks <algm-ClipPeaks>` to clip peaks, providing a background estimation.
+- :ref:`PolDiffILLReduction <algm-PolDiffILLReduction>` to perform polarised diffraction data reduction for the ILL D7 instrument
+- :ref:`D7AbsoluteCrossSections <algm-D7AbsoluteCrossSections>` to separate magnetic, nuclear coherent, and incoherent cross-sections using spin-flip and non-spin-flip cross-sections, and to normalise D7 data to a given standard
+
+.. figure:: ../../images/IV_D1B.png
+   :class: screenshot
+   :width: 400px
+   :align: right
+
+Improvements
 ############
 
-- Add ability to store multiple alternative attenuation file paths in the Pearl yaml configuration file
-- Modify filenames of xye outputs from running a focus in the Pearl power diffraction scripts
-- Remove _noatten workspace that was produced by the Pearl powder diffraction scripts when run with perform_attenuation=True
+.. figure:: ../../images/Bk2Bk_exp.png
+   :class: screenshot
+   :width: 500px
+   :align: right
+
+- **Files from ILL's instrument D1B can now be loaded**
```

Comment:
I suggest adding the word "Nexus" to this so it reads "Nexus files from ILL's instrument..."
Also it wasn't totally obvious which items the screenshots relate to. I suggest you put "(see screenshot of instrument view)" or similar on the end of this line.